Atlas lcmtypes error when building w/o atlas_drivers external

I am trying to build OH without access to the atlas_drivers external (but have access to SNOPT and built gurobi on its own) and am getting the following errors:

> [ 14%] Building CXX object src/CMakeFiles/AtlasFallDetector.dir/AtlasFallDetector.cpp.o
> In file included from /home/rc/oh-distro/software/control/src/AtlasFallDetector.cpp:1:0:
> /home/rc/oh-distro/software/control/src/AtlasFallDetector.hpp:12:39: fatal error: lcmtypes/atlas/status_t.hpp: No such file or directory
>  #include "lcmtypes/atlas/status_t.hpp"
>                                        ^
> compilation terminated.
> make[4]: *** [src/CMakeFiles/AtlasFallDetector.dir/AtlasFallDetector.cpp.o] Error 1
> make[3]: *** [src/CMakeFiles/AtlasFallDetector.dir/all] Error 2
> make[2]: *** [all] Error 2
> make[1]: *** [all] Error 2
> make: *** [all] Error 2

I tried cd'ing into drc_lcmtypes, doing a make clean, then make again in /software. That didn't work.

--- a/software/atlas-collection/CMakeLists.txt
+++ b/software/atlas-collection/CMakeLists.txt
@@ -46,10 +46,14 @@ macro(add_external_project proj)
 endmacro()
 
 
+add_external_project(atlas_lcmtypes)
+checkbuild(atlas_lcmtypes)
+
+add_external_project(bdi_walking)
+checkbuild(bdi_walking)
+
 # Only build if private modules atlas and flycapture have been checked out
 if(BUILD_ATLAS_DRIVER)
-  add_external_project(atlas_lcmtypes)
-  checkbuild(atlas_lcmtypes)
 
   set(atlas_depends atlas_lcmtypes)
   add_external_project(atlas)
@@ -58,9 +62,6 @@ if(BUILD_ATLAS_DRIVER)
   set(state_sync_atlas_depends atlas atlas_lcmtypes)
   add_external_project(state_sync_atlas)
   checkbuild(state_sync_atlas)
-
-  add_external_project(bdi_walking)
-  checkbuild(bdi_walking)
 
   add_external_project(flycapture)
   checkbuild(flycapture)

